ui/pages/destinations/add.js

```diff
@@ -2,8 +2,10 @@ import Head from 'next/head'
 import Link from 'next/link'
 import { useState, useEffect } from 'react'
 import copy from 'copy-to-clipboard'
+import yaml from 'js-yaml'
```

Comment:
JSON is valid yaml, so we could potentially avoid this dependency by using `JSON.stringify()` and writing it out as JSON.

Reply:
We could do that but IMO JSON is less user friendly with its quotes and braces and brackets. I don't think it's worth using JSON solely to avoid this dependency.